src/serialization.js

```diff
@@ -0,0 +1,91 @@
+export const noop = (payload) => payload;
+
+const transformPayload = (message, transformer = noop) => ({
```

Comment:
Might be good to mention here why a new object is required rather than mutating the message. It took me a second to realize that manipulations here would manipulate the original action object which could cause unintentional side-effects.

Reply:
Will add a comment.